src/test/java/org/simulator/sbml/SBMLTestSuiteRunner.java

```diff
@@ -969,7 +969,7 @@ public void testModels() throws FileNotFoundException, IOException {
             boolean errorInSolve = false;
             try {
               solution = solver.solve(interpreter,
-                interpreter.getInitialValues(), timepoints);
+                interpreter.getInitialValues(), timepoints, null);
             } catch (DerivativeException e) {
               errorInSolve = true;
```

Comment:
Please remove this and throw a "normal" exception. We want to know why the error occurs.